web/template/admin/lecture-cut.gohtml

```diff
@@ -4,10 +4,10 @@
     <meta charset="UTF-8">
     <title>TUM Live | Edit Units</title>
     {{template "headImports"}}
-    <script>window.HELP_IMPROVE_VIDEOJS = false;</script>
-    <link rel="stylesheet" href="/static/node_modules/video.js/dist/video-js.min.css">
+    <script>window.HELP_IMPROVE_watchJS = false;</script>
```

Comment:
```suggestion
    <script>window.HELP_IMPROVE_VIDEOJS = false;</script>
```

Reply:
Yikes :disappointed:  Nice catch! Thanks for pointing these out, not sure how I overlooked this, it happened when I changed the library name, I think. :see_no_evil:

Reply:
Good old refactor -> rename :D